spec/src/main/asciidoc/optional-container-integration.asciidoc

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2020 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// Ladislav Thon
+
+[[optional-container-integration]]
+
+== Recommendations for Optional Container Integration
+
+This section describes the expected behaviors when the implementation runs in a Java EE container.
```

Comment:
Java EE or Jakarta EE

Reply:
We refer to Java EE elsewhere in the spec, so I decided to stick to that.

Reply:
The 'Java EE or Jakarta EE' is tangential to this PR.

However, there appears to be only 3 mentions of 'Java EE' though my understanding is all occurrences should be ultimately renamed to Jakarta EE in a subsequent PR.

Reply:
I agree this should be all renamed to Jakarta EE, but not in this PR.

Reply:
Filed #556 for this.